integration/benchnet2/automate/templates/helm-values-all-nodes.yml

```diff
@@ -0,0 +1,385 @@
+        - --profiler-enabled=false
+        - --profile-uploader-enabled=false
+        - --tracer-enabled=false
+        - --bind=0.0.0.0:3569
+        - --profiler-dir=/profiler
+        - --profiler-interval=2m
+        - --nodeid={{$val.node_id}}
+        - --block-rate-delay=800ms
+        - --hotstuff-timeout=2s
+        - --hotstuff-min-timeout=2s
+        - --chunk-alpha=1
+        - --emergency-sealing-active=false
+        - --insecure-access-api=false
+        - --access-node-ids=*
+      env:
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: http://tempo:4317
+        - name: OTEL_EXPORTER_OTLP_TRACES_INSECURE
+          value: "true"
+        - name: OTEL_RESOURCE_ATTRIBUTES
+          value: network
+        - name: BINSTAT_ENABLE
+          value: ""
+        - name: BINSTAT_LEN_WHAT
+          value: ""
+        - name: BINSTAT_DMP_NAME
+          value: ""
+        - name: BINSTAT_DMP_PATH
+          value: ""
+      image: gcr.io/flow-container-registry/consensus:v0.27.6
+      nodeId: {{$val.node_id}}
+{{end}}{{end}}
+execution:
+  defaults:
+    imagePullPolicy: Always
+    containerPorts:
+      - name: ptp
+        containerPort: 3569
+      - name: grpc
+        containerPort: 9000
+      - name: secure-grpc
+        containerPort: 9001
+      - name: admin
+        containerPort: 9002
+    env: []
+    resources:
+      requests:
+        cpu: "200m"
+        memory: "512Mi"
+    servicePorts:
+      - name: ptp
+        protocol: TCP
+        port: 3569
+        targetPort: ptp
+      - name: grpc
+        protocol: TCP
+        port: 9000
+        targetPort: grpc
+      - name: secure-grpc
+        protocol: TCP
+        port: 9001
+        targetPort: secure-grpc
+      - name: admin
+        protocol: TCP
+        port: 9002
+        targetPort: admin
+    storage: 10G
+  nodes:
+{{range $val := .}}{{if eq ($val.role) ("execution")}}
+    {{$val.name}}:
+      args:
+        - --bootstrapdir=/bootstrap
+        - --datadir=/data/protocol
+        - --secretsdir=/data/secret
+        - --loglevel=INFO
+        - --bind=0.0.0.0:3569
+        - --profiler-enabled=false
+        - --profile-uploader-enabled=false
+        - --tracer-enabled=false
+        - --profiler-dir=/profiler
+        - --profiler-interval=2m
+        - --nodeid={{$val.node_id}}
+        - --triedir=/trie
+        - --rpc-addr=0.0.0.0:9000
+        - --cadence-tracing=false
+        - --extensive-tracing=false
+      env:
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: http://tempo:4317
+        - name: OTEL_EXPORTER_OTLP_TRACES_INSECURE
+          value: "true"
+        - name: OTEL_RESOURCE_ATTRIBUTES
+          value: network
+        - name: BINSTAT_ENABLE
+          value: ""
+        - name: BINSTAT_LEN_WHAT
+          value: ""
+        - name: BINSTAT_DMP_NAME
+          value: ""
+        - name: BINSTAT_DMP_PATH
+          value: ""
+      image: gcr.io/flow-container-registry/execution:v0.27.6
```

Comment:
We will want to support the templating of the container tag. A variable should be injected so that this can be templated.

This should be the case for all `image` variables in all of the node definitions.

Reply:
Updated support for Docker image tags in b3c91ceb01e1173756ac1178c1adc84fffc8bd97, f8098eece82da95c6bb4730191dae699c5f948ad, b5b1ef52db5b16ef0ec71a17be42f354ea0f115c